apps/console/src/features/applications/pages/applications.tsx

```diff
@@ -297,7 +297,7 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
             const appList: ApplicationListInterface = cloneDeep(applicationList);
 
             appList.applications = appList.applications.filter((item: ApplicationListItemInterface) =>
-                !ApplicationManagementConstants.SYSTEM_APPS.includes(item.clientId)
+                !ApplicationManagementConstants.HIDDEN_APPS.includes(item.name)
```

Comment:
Any reason not to use client id?

Reply:
Client id is not consistent for sub orgs and super orgs. Since application name is unique currently it is safe. But these things should be fixed from backend. Once fixed we have to change the UI logic. Ref: https://github.com/wso2/identity-apps/blob/00a3522ae1c8cbca22317ffecd6c2ad710fa35b1/apps/console/src/features/applications/pages/applications.tsx#L288-L293

Reply:
Ack.